README.md

```diff
@@ -21,6 +21,7 @@ Join us if you dare!  This is not for the faint of heart!
 
 * [Setup](manuscript/setup.md)
 * [Build Container Image](manuscript/build-container-image/story.md)
+* [Store Container Image in a Registry](manuscript/registry/story.md)
```

Comment:
Sounds good, I trust your judgment.  

Another idea is to make a "Skip Ahead" section with links to each step of the story.

Reply:
"Skip ahead" would be difficult if one is supposed to follow along since steps often depend on previous choices. For example, if you haven't built an image, there's nothing to push to a registry.

Reply:
You're right.

It'll be a pita to get back to the appropriate spot if you quit before you're finished.  Not sure how to solve that.